### DISCUSSION TITLE
0 EPS, NO alerts

### DISCUSSION THREAD

**Discussion by oliverhavrila** [Security-Onion-Solutions/securityonion#12137] [2.4]
### Version

2.4.30

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

2 x 14 Core 2.40 GHz

### RAM

128

### Storage for /

265G SSD

### Storage for /nsm

6,5T SSD

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

Dear,

I would like to ask you... month ago we been discussing about that so-elastalert is not working. After your recomendetation I did fresh installation.
But after few days later, No new alerts are creating, 0 EPS, but inboud traffic at this moment for example is about 1,4 Mb/s

Any advice, how I can check whats happend?

Thank you

sudo salt-call state.highstate - show me: Succeeded: 781 (changed=43)  Failed:      2
Failed to query for current Logstash Outputs...
Failed to query for current Fleet Server Elasticsearch URLs...

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by cm-ops** [comment]
What does `elastic-agent status` show?
